src/claim/Claim.utils.ts

```diff
@@ -4,10 +4,129 @@
  * @preferred
  */
 
+import { AnyJson } from '@polkadot/types/types'
+import { v4 as uuid } from 'uuid'
+import { hexToBn } from '@polkadot/util'
 import jsonabc from '../util/jsonabc'
 import * as SDKErrors from '../errorhandling/SDKErrors'
 import IClaim, { CompressedClaim } from '../types/Claim'
 import { validateAddress, validateHash } from '../util/DataUtils'
+import { getIdForCTypeHash } from '../ctype/CType.utils'
+import { HashingOptions, hashStatements } from '../crypto/Crypto'
+
+const VC_VOCAB = 'https://www.w3.org/2018/credentials#'
+
+/**
+ * The minimal partial claim from which a JSON-LD representation can be built.
+ */
+export type PartialClaim = Partial<IClaim> & Pick<IClaim, 'cTypeHash'>
+
+function JsonLDcontents(
+  claim: PartialClaim,
+  expanded = true
+): Record<string, AnyJson> {
+  const { cTypeHash, contents, owner } = claim
+  if (!cTypeHash)
+    throw new Error('ctype hash is required for conversion to json-ld')
```

Comment:
please also provide an error according to error handler here

Reply:
Here you still not use our error workflow

Reply:
ooooupsie